trade-finance-manager-ui/templates/case/_macros/save-only-button.njk

```diff
@@ -0,0 +1,16 @@
+# This macro renders primary `Save and close` button only
```

Comment:
Maybe a typo -  file is named `save-only-button`, should it be `save-and-close-button`?